assets/pages/bundles-landing/helpers/abtest.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+// ----- Imports ----- //
+
+import type { Participation } from 'helpers/abtest';
+
+
+// ----- Functions ----- //
+
+const otherWaysOfContribute = (participation: Participation): string => {
+
+  const variant = participation.otherWaysOfContribute;
+
+  switch (variant) {
+    case 'control' : return 'other ways you can support us';
+    case 'variantA' : return 'other ways you can contribute';
+    case 'variantB' : return 'other ways you can give us money';
```

Comment:
This is a good point.  I think we will need to do some refactoring of the a/b test framework as we run the first few tests.  The aim should be to need the minimum amount of work for each a/b test.